S/SimpleITK/build_tarballs.jl

```diff
@@ -0,0 +1,78 @@
+using BinaryBuilder, Pkg
+
+name = "SimpleITK"
+version = v"2.2.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("http://github.com/SimpleITK/SimpleITK/releases/download/v$(version)/SimpleITK-$(version).tar.gz", "b07bb98707556ebc2b79aac22dc14950749f509e5b43da8043233275aa55488a")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+mount -t tmpfs -o size=12G tmpfs /workspace/srcdir
+cd ..
+cd ..
+cd workspace/srcdir
+wget https://github.com/SimpleITK/SimpleITK/releases/download/v2.2.0/SimpleITK-2.2.0.tar.gz
+tar -xvzf ./SimpleITK-2.2.0.tar.gz 
+mkdir SimpleITK-build
+cd SimpleITK-build/
+cmake -DCMAKE_INSTALL_PREFIX:FILEPATH=/workspace/destdir -DCMAKE_BUILD_TYPE:STRING=RELEASE -DBUILD_SHARED_LIBS:BOOL=ON ../SimpleITK-2.2.0/SuperBuild
+make -j${nproc}
+rm -rf /workspace/srcdir/SimpleITK-build/ITK-build/lib/cmake
+cd SimpleITK-build
+make install
+cp /workspace/srcdir/SimpleITK-build/ITK-build/lib/* /workspace/destdir/lib
+cd /workspace/destdir/share
+mkdir licenses
+cd licenses
+mkdir SimpleITK
+cd SimpleITK
+cp /workspace/destdir/share/doc/SimpleITK-2.2/* ./ 
```

Comment:
The binarybuilder couldnt find a valid license since the directory structure was in the following format 
/workspace/destdir/share/doc/SimpleITK-2.2/ and stuff
so i had to make a licenses directory in the share directory with a valid license for bianrybuilder to find.

Reply:
Please [read the documentation](https://docs.binarybuilder.org/stable/build_tips/#Installing-the-license-file).  You don't create the directory manually, you use the utility we provide for this purpose.